juniper/src/types/async_await.rs

```diff
@@ -258,31 +258,29 @@ where
                 let is_non_null = meta_field.field_type.is_non_null();
 
                 let response_name = response_name.to_string();
-                async_values.push_back(AsyncValueFuture::Field(async move {
-                    // TODO: implement custom future type instead of
-                    //       two-level boxing.
```

Comment:
I don't see any boxing at all.. `AsyncValueFuture` is generic so this comment looks outdated.